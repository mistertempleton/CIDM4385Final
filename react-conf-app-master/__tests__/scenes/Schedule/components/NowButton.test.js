// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import NowButton from '../../../../app/scenes/Schedule/components/NowButton';

describe('Schedule - Now Button', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NowButton onPress={() => {}} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
