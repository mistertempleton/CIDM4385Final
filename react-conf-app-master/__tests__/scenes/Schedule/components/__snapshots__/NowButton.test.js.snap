exports[`Schedule - Now Button renders correctly 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "left": 0,
      "paddingBottom": 20,
      "position": "absolute",
      "right": 0,
      "transform": Array [
        Object {
          "scale": 1,
        },
      ],
    }
  }>
  <BVLinearGradient
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    colors={
      Array [
        4279217626,
        4278230731,
      ]
    }
    end={undefined}
    hitSlop={
      Object {
        "bottom": 20,
        "left": 10,
        "right": 10,
        "top": 10,
      }
    }
    locations={null}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    start={undefined}
    style={
      Object {
        "backgroundColor": "#00A8D8",
        "borderRadius": 40,
        "height": 40,
        "justifyContent": "center",
        "paddingHorizontal": 32,
        "shadowColor": "black",
        "shadowOffset": Object {
          "height": 1,
          "width": 0,
        },
        "shadowOpacity": 0.24,
        "shadowRadius": 2,
      }
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "backgroundColor": "transparent",
          "color": "white",
          "fontWeight": "bold",
        }
      }>
      NOW
    </Text>
  </BVLinearGradient>
</View>
`;
